src/components/Forgot/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Alert } from "antd";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+
+function Forgot({ isTesting }) {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const forgotPassword = (e) => {
+    setErrorMessage(null);
+    firebase
+      .auth()
+      .sendPasswordResetEmail(email)
+      .then(function (user) {
+        alert("Please check your email...");
```

Comment:
lets not use the alert function here it doesn't look professional maybe there's a component in antd for alerts try to use this one

Reply:
https://ant.design/components/message/#header
https://ant.design/components/alert/#header

Reply:
Can you use the links I mentioned above for the alerts, use antd message maybe? @gizemcdeniz